chapters/appendix.adoc

```diff
@@ -60,7 +60,7 @@ Chapter <<Offenses>>, section <<Minor Offenses>>:
 | Event | Applicability | Command | AutoRef
 
 | <<Aimless Kick [small]#(_division B only_)#, Aimless Kick>> | <<Ball In And Out Of Play, ball in play>> | <<Stop>>, then <<Indirect Free Kick>> | icon:check[role="green"]
-| <<Lack Of Progress>> | <<Ball In And Out Of Play, ball in play>> | <<Stop>>, then <<Indirect Free Kick>> | icon:check[role="green"]
+| <<Lack Of Progress>> | <<Ball In And Out Of Play, ball in play>> | <<Stop>>, then <<Force Start>> | icon:check[role="green"]
```

Comment:
Force Start doesn't work if the ball is inside the defense Area.

Reply:
just like free kick. But as the game is stopped before the next command, there will also be a ball placement, if the ball is not at valid position.

Reply:
It is well-defined what happens if a free-kick is given at an invalid position (as far as i remember).

I do not remember an analogous rule regarding force start. I might be wrong, though, but that's the reason I mentioned it here.

Reply:
I've found it. It's referenced in this document (see xxx for details regarding the positon), and is explained here:
```

Definition

The ball placement position for a free kick depends on the event that led to the free kick. This position is valid if there is at least 0.2 meters distance to all field lines and 1 meter distance to either defense area. If an event requires the ball to be placed at a position that contravenes this rule, it has to be placed at the closest valid position instead.
```

There is no rule how to handle the same situation for a force start.
If there's a ball placement into a valid position, we should note that here.
Also, is ballplacement into force-start already a valid transition? Because it feels to me like you'd have a huge benefit if you're getting to place the ball as you might be much closer than your opponent after the ballplacement.

If you have to drive away it should be faster to resume the game via an indirect free kick than via force start. Sadly, we dumped Indirect free kicks so now we're lost as a direct free kick is a much stronger tool in this situation :(

Reply:
as far as I read the rules, there is no specific definition for how the ball is placed for free kicks. It just says, the ball must be placed after stoppage and there is the condition with the penalty area.

Reply:
oh, you were faster. And I actually misread the rules. I thought, the distance to the defense area is defined in the ball placement section.

Reply:
oh wait, but lack of progress can only happen from free kick, kick off and penalty kick positions. With the old penalty kick, there might be an issue with the position, but with the new procedure, all positions should already be fine.
Additionally, the whole idea of the force start was actually to not stop the game in between. Otherwise, we would not really save time with the force start.

Reply:
This whole discussion is centered around the lack of progress inside the defense area, if the keeper is unable to clear the ball in 5 seconds...

Reply:
@rhololkeolke I think that still needs to be specified somewhere. The force start is only mentioned in the appendix and the usual procedure is to first stop the game.

Reply:
oh, now I see. It is an additional sentence in the rules. Would be better to attach the comment directly to this line. I'll do it.